web/html/src/manager/channels/subscribe-channels/subscribe-channels.js

```diff
@@ -97,17 +97,22 @@ class SystemChannels extends React.Component<SystemChannelsProps, SystemChannels
   updateView = () => {
     Network.get(`/rhn/manager/api/systems/${this.props.serverId}/channels`)
       .promise.then(data => {
-        const base : ChannelDto = data.data && data.data.base ? data.data.base : this.getNoBase();
+      const base : ChannelDto = data.data && data.data.base ? data.data.base : this.getNoBase();
```

Comment:
Can you please re-adjust the indentation here for the whole block? It should look like
```
.promise.then(data=> {
  const base ...........
```

instead of
```
.promise.then(data=> {
const base ...........
```